db/version_set.cc

```diff
@@ -2006,6 +2006,17 @@ void Version::MultiGet(const ReadOptions& read_options, MultiGetRange* range,
           }
           PERF_COUNTER_BY_LEVEL_ADD(user_key_return_count, 1,
                                     fp.GetHitFileLevel());
+          file_range.AddValueSize(iter->value->size());
+          if (file_range.GetValueSize() > read_options.value_size_soft_limit) {
+            // Set all remaining keys in range to Abort
+            for (auto range_iter = range->begin(); range_iter != range->end();
+                 ++range_iter) {
+              range->MarkKeyDone(range_iter);
+              *(range_iter->s) = Status::Aborted();
+            }
+            iter = file_range.end();
```

Comment:
Can we set ```s = Status::Aborted()``` and use that to break out of the nested loops? Or maybe checking for ```file_range.empty()``` would work, like in line 2045.